pandas/core/series.py

```diff
@@ -5629,6 +5636,129 @@ def between(
 
         return lmask & rmask
 
+    def case_when(
```

Comment:
Could you add this to the API reference too?

Reply:
@mroeschke where is the API reference located?